File: SignatureRequests/SignatureRequests.Core/RequestObjects/BoxRequest.cs
using SignatureRequests.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SignatureRequests.Core.RequestObjects
{
    public class BoxRequest
    {
        public int Id { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }
        public int X { get; set; }
        public int Y { get; set; }
        public string Type { get; set; }
        public string SignerType { get; set; }
        public string SignedStatus { get; set; }
        public RequestRequest Request { get; set; }
        public int? RequestId { get; set; }
        public SignatureEntity Signature { get; set; }
        public int? SignatureId { get; set; }
        public FormRequest Form { get; set; }
        public int FormId { get; set; }
        public int PageNumber { get; set; }
        public bool IsModel { get; set; }
        public string Text { get; set; }
        public DateTime? Date { get; set; }

    }
}

File: SignatureRequests/SignatureRequests.Webs/Scripts/react/src/Entities/BoxRequest.ts

class BoxRequest {
  width: number;
  height: number;
  x : number;
  y: number;
  type: string;
  signerType: string;
  signedStatus: string;
  requestId?: number;
  signatureId?: number;
  formId?: number;
  pageNumber: number;
  isModel: boolean;
  text?: string;
  date?: Date;
  formHeight: number;
  formWidth: number;

  constructor(data: any) {
    this.width = data.width;
    this.height = data.height;
    this.x = data.x;
    this.y = data.y;
    this.type = data.type;
    this.signerType = data.signerType;
    this.signedStatus = data.signedStatus;
    this.requestId = data.requestId;
    this.signatureId = data.signatureId;
    this.formId = data.formId;
    this.pageNumber = data.pageNumber;
    this.isModel = data.isModel;
    this.text = data.text;
    this.date = data.date;
    this.formHeight = data.formHeight;
    this.formWidth = data.formWidth;
  }
}

export default BoxRequest;
